tools/azure-pipelines/jobs-template.yml

```diff
@@ -199,7 +199,7 @@ jobs:
       condition: not(eq(variables['SKIP'], '1'))
     - task: Cache@2
       inputs:
-        key: e2e-cache | flink-end-to-end-tests/**/*.java, !**/avro/**
```

Comment:
what is this for?

Reply:
I believe the e2e cache never really worked, as it included only *.java files.

Reply:
But isn't this line just a `key` to determine whether a cache should be used, and not an inclusion filter?

Reply:
gosh, not sure what's going on with my brain. You are right.